### DISCUSSION TITLE
OpenApi specs yaml/json

### DISCUSSION THREAD

**Discussion by paul-asvb** [teamhanko/hanko#960] [Q&A] [answered]
Hello hanko team,
where could anyone point me to the openapi/swagger specification yaml/json?
redocly is using it i guess.
greetings

**Comment by lfleischmann** [comment, marked as answer]
Hi @paul-asvb,

you can find the specs [here](https://github.com/teamhanko/hanko/tree/main/docs/static/spec).

**Comment by paul-asvb** [comment]
Thank you
